src/content/en/2021/caching.md

```diff
@@ -1,20 +1,489 @@
+jessnicolet_bio: Jessica began her career as an opera singer and has been in the classical music industry for the past 10 years. In early 2020 and due to the pandemic, she decided to start a new career in Tech, specifically Web Development. She has always enjoyed writing and telling stories both onstage and off and published a [series of three articles](https://jessicanicolet.medium.com/) on Medium documenting her experience transitioning to this new field. She is currently looking for a full-time position in Technical Writing.
 results: https://docs.google.com/spreadsheets/d/1-v3yR0LZIC3t4zWtqTgR3jJsKjjRMP-HATU2caP8e2c/
-featured_quote: TODO
-featured_stat_1: TODO
-featured_stat_label_1: TODO
-featured_stat_2: TODO
-featured_stat_label_2: TODO
-featured_stat_3: TODO
-featured_stat_label_3: TODO
+featured_quote: Adoption of cache headers continues to steadily grow as we learn to navigate the multiverse of caching choices. Optimize pages not only for HighEnd devices but also for the next billion users that access your product from LowEnd devices.
+featured_stat_1: 99.8%
+featured_stat_label_1: Pages caching fonts
+featured_stat_2: 51 trillion year
+featured_stat_label_2: Largest `max-age` value recorded
+featured_stat_3: 59.1%
+featured_stat_label_3: Mobile pages could save up to 1 MB with caching
+unedited: true
 ---
 
-## TODO
+## Introduction
+
+Over the last two decades, the way we experience web applications has changed, giving us richer and more interactive content. But, unfortunately, this content comes with a cost in both data storage and bandwidth. Most of the time, this makes it harder for many of us to fully experience a web product when the network we use is degraded, or our device doesn't have enough space. Caching is both a solution to and the cause of some of these problems. Learning to navigate the multiverse of choices will enable you to build not only for *HighEnd* devices but also for the next billion users that access your product from *LowEnd* devices.
+
+Caching is a technique that enables the reuse of previously downloaded content, from simple static assets like Javascript, CSS files or basic string values to more complex JSON API responses.
+
+At its core, Caching avoids making specific HTTP requests and allows an application to feel more responsive and reliable to the user. Each request is usually cached in two main places:
+- **Content Delivery Network (CDN)**: Usually a third-party company like Cloudflare with the primary goal of replicating your data as closely as possible to where the user is accessing the application. Most CDNs have some default behavior, but mainly you can give them instructions on how to cache by using HTTP Headers.
```

Comment:
Edit: 

- **[Content Delivery Network (CDN)](https://20211129t162816-dot-webalmanac.uk.r.appspot.com/en/2021/cdn)**: Usually a third-party company with the primary goal of replicating your data as closely as possible to where the user is accessing the application. Most CDNs have some default behavior, but mainly you can give them instructions on how to cache by using HTTP Headers.